src/components/shared/page.tsx

```diff
@@ -82,13 +82,18 @@ interface BodyProps {
     capi: Content;
 }
 
+interface ElementWithScript {
+    element: JSX.Element;
```

Comment:
`React.ReactElement`?

Reply:
Will give that a try, it may of been a legacy type used.